src/js/components/wizard/service-hours.js

```diff
@@ -1,115 +1,114 @@
 /*eslint-disable no-unused-vars*/
 import React from 'react';
 import DropDown from '../drop-down';
-import { RaisedButton, CardText, FontIcon, TimePicker } from 'material-ui';
+import { RaisedButton, FlatButton, CardText, FontIcon, TimePicker, SelectField, MenuItem } from 'material-ui';
+import { HoursTable } from '../hours-table';
 /*eslint-enable no-unused-vars*/
 
-import { bindAll } from 'lodash';
 import WizardPage from './wizard-page';
+import { Schedule, days, nextDay } from 'btc-models';
 
-const weekDays = [
-  'Weekdays', 'Weekends', 'Monday', 'Tuesday',
-  'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'
-];
+import { bindAll, keys, toPairs } from 'lodash';
+
+// The pickers default to 8 AM and 5 PM respectively
+const defaultOpens = new Date( 1970, 0, 1, 8, 0, 0, 0 );
+const defaultClose = new Date( 1970, 0, 1, 17, 0, 0, 0 );
 
 export class ServiceHours extends WizardPage {
   constructor( props ) {
     super( props );
-    bindAll( this, 'onDaySelect', 'addHours', 'removeHours' );
+    bindAll( this, 'addHours', 'removeHours', 'onRowSelection' );
```

Comment:
Binds instance methods to this, just like this.addHours = this.addHours.bind( this ). Its good to use when your class has a lot of event handlers.